libmamba/include/mamba/core/pool.hpp

```diff
@@ -22,6 +22,64 @@ namespace spdlog
 
 namespace mamba
 {
+    class MQueue
+    {
+    public:
+        MQueue()
+            : p_queue(new Queue)
+        {
+            queue_init(p_queue);
+        }
+
+        ~MQueue()
```

Comment:
This deletes copy and move operators, although for future readers I think it would be best to state that explicitly, as intentional (with `= delete`).

Reply:
Good idea. We can also implement them (now or later).